environments/dev-env/CONTRIBUTING.md

```diff
@@ -0,0 +1,510 @@
+
+  ```shell
+  kcp start --bind-address <ip-address> >& kcp_log.txt &
+  ```
+  
+  This uses default 6443 port, just bind to a specific address
+  
+  Set your KUBECONFIG variable to the file in `.kcp/admin.kubeconfig` generated by kcp during startup:
+
+  ```shell
+  export KUBECONFIG="$(pwd)/.kcp/admin.kubeconfig"
+  ```
+
+  c) Deploy kcp-edge infra:
+         
+  Run the following command inside the `kcp-edge/environments/dev-env` directory:
+
+  ```bash
+  ./kcp-edge.sh start
+  ```
+
+  The following components are created:
+
+  - 1 kcp workspace: edge service provider workspace (`espw`)
+
+  ```console
+  kubectl ws tree
+  .
+  └── root
+      ├── compute
+      └── espw
+  ```
+
+  - 3 kcp-edge controllers: [edge-scheduler](https://docs.kcp-edge.io/docs/coding-milestones/poc2023q1/edge-scheduler/), [mailbox-controller](https://docs.kcp-edge.io/docs/coding-milestones/poc2023q1/mailbox-controller/) and [placement-translator](https://docs.kcp-edge.io/docs/coding-milestones/poc2023q1/placement-translator/)
+
+  ```shell
+  ps aux | grep -e mailbox-controller -e placement-translator -e scheduler
+  user    11344   0.1  0.1 34906508  28044 s001  S     8:36PM   0:00.43 placement-translator --allclusters-context system:admin -v=2
+  user    11333   0.1  0.1 34885544  19536 s001  S     8:36PM   0:00.38 scheduler -v 2 --root-user kcp-admin --root-cluster root --sysadm-context system:admin --sysadm-user shard-admin
+  user    11323   0.0  0.1 34892244  20780 s001  S     8:36PM   0:00.13 mailbox-controller --inventory-context=root --mbws-context=base -v=2
+  ```
+
+### 5. Connect your edge cluster to the kcp-edge platform:
+
+  a) Create inventory management workspace (`imw`):
+
+  ```console
+  kubectl ws root
+  kubectl ws create imw-1 --enter
+
+  Workspace "imw-1" (type root:organization) created. Waiting for it to be ready...
+  Workspace "imw-1" (type root:organization) is ready to use.
+  Current workspace is "root:imw-1" (type root:organization).
+  ```
+    
+  b) Create a syncTarget and location objects to represent your edge cluster (florin):
+
+  ```console
+  ensure-location.sh florin  env=prod
+
+  synctarget.workload.kcp.io/florin created
+  location.scheduling.kcp.io/florin created
+  synctarget.workload.kcp.io/florin labeled
+  location.scheduling.kcp.io/florin labeled
+  ```
+
+  A location and synctarget objects are created:
+
+  ```console
+  kubectl get locations,synctargets
+  NAME                                RESOURCE      AVAILABLE   INSTANCES   LABELS   AGE
+  location.scheduling.kcp.io/florin   synctargets   0           1                    57s
+
+  NAME                                AGE
+  synctarget.workload.kcp.io/florin   58s
+  ```
+
+
+  The [mailbox-controller](https://docs.kcp-edge.io/docs/coding-milestones/poc2023q1/mailbox-controller/) creates a mailbox workspace for the newly created SyncTarget: `florin`:
```

Comment:
I think setting "`florin`" off with a colon says that `florin` is the mailbox workspace for the newly created SyncTarget.  I think it would be clearer to put `florin` in parentheses right after "SyncTarget".

Reply:
fixed